docs/manage/ml/train-model.md

```diff
@@ -10,37 +10,26 @@ description: "Train an image classification model on labeled image data."
 # SME: Aaron Casas
 ---
 
-You can label or add bounding boxes to [images collected](/services/data/configure-data-capture/) by robots and use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model** or **Object Detection Model** within Viam.
-
-{{<youtube embed_url="https://www.youtube-nocookie.com/embed/CP14LR0Pq64">}}
+You can label or add bounding boxes to [images collected](/services/data/configure-data-capture/) by robots, add them to a dataset, and use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model** or **Object Detection Model** within Viam.
```

Comment:
Labels are comprised of image classification tags and bounding boxes, so would either have this be 'you can label images collected' or 'you can add classification tags or bounding boxes to images collected'

Reply:
```suggestion
You can add classification tags or bounding boxes to [images collected](/services/data/configure-data-capture/) by robots, add them to a dataset, and use the annotated data to train a **Single Label Classification Model**, **Multi Label Classification Model** or **Object Detection Model** within Viam.
```